app/src/pages/TopicsDetails.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { toRefs } from "vue";
+import { db, DocType, type Uuid } from "luminary-shared";
+import { appLanguageIdAsRef } from "@/globalConfig";
+import { useRouter } from "vue-router";
+import ContentTile from "@/components/content/ContentTile.vue";
+
+const router = useRouter();
+
+const { id } = toRefs(router.currentRoute.value.params);
+
+const parentTopic = db.whereParentAsRef(id.value as Uuid, DocType.Tag, appLanguageIdAsRef.value);
+
+const relatedContent = db.contentWhereTagAsRef(id.value as Uuid, {
+    languageId: appLanguageIdAsRef.value,
+});
+</script>
```

Comment:
Perhaps set the page name here, similar to what we do in `SingleContent` for posts? So that it reads `Topic A - {appName}`